chat-widget/src/common/telemetry/TelemetryConstants.ts

```diff
@@ -48,7 +48,10 @@ export enum BroadcastEvent {
     ClosePopoutWindow = "ClosePopoutWindow",
     RaiseErrorEvent = "RaiseErrorEvent",
     NetworkDisconnected = "NetworkDisconnected",
-    NetworkReconnected = "NetworkReconnected"
+    NetworkReconnected = "NetworkReconnected",
+    SigninCardReceived = "lcw:signInCardReceived",
```

Comment:
Let’s just use “SigninCardReceived” and not start with “lcw”, Asian parity with other events

Reply:
done